<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="IN-Sandeep" name="Test" tests="2" failures="1" timestamp="17 Jul 2017 10:57:52 GMT" time="0.046" errors="0">
  <testcase name="@BeforeSuite init" time="0.0" classname="DD_TestCases.LoginTest">
    <failure type="java.io.FileNotFoundException" message="C:\SeleniumCoaching\DataDrivenFramework\src\main\java\DD_Config\Config.properties (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\SeleniumCoaching\DataDrivenFramework\src\main\java\DD_Config\Config.properties (The system cannot find the path specified)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at java.io.FileInputStream.<init>(FileInputStream.java:93)
	at DD_Core.Page.init(Page.java:40)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeSuite init -->
  <testcase name="@BeforeTest isSkip" time="0.0" classname="DD_TestCases.LoginTest">
    <skipped/>
  </testcase> <!-- @BeforeTest isSkip -->
  <testcase name="doLogin" time="0.002" classname="DD_TestCases.LoginTest">
    <skipped/>
  </testcase> <!-- doLogin -->
  <testcase name="doLogin" time="0.028" classname="DD_TestCases.LoginTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at DD_Util.Commons.captureScreenshot(Commons.java:46)
	at DD_Util.TestListener.onTestSkipped(TestListener.java:51)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- doLogin -->
</testsuite> <!-- Test -->
